tempesta_fw/http.c

```diff
@@ -2120,6 +2120,27 @@ tfw_http_set_loc_hdrs(TfwHttpMsg *hm, TfwHttpReq *req)
 	return 0;
 }
 
+static int
+tfw_http_recreate_content_type_multipart_hdr(TfwHttpReq *req)
+{
+	TfwStr replacement = {
+		.ptr = (TfwStr []) {
+			TFW_STR_FROM("Content-Type"),
+			TFW_STR_FROM(": "),
+			TFW_STR_FROM("multipart/form-data; boundary="),
+			req->multipart_boundary_raw,
```

Comment:
`req->multipart_boundary_raw` can't be compound string according to the usage. This should be asserted in unit tests, but i didn't found such check.  But as I see usage of the variable in the http_parser, it actually is compound.

Reply:
Indeed, `req->multipart_boundary_raw` can be compound string, and I totally forgot about that. Fixed by explicitly handling both cases of compound and plain types of `req->multipart_boundary_raw`.